packages/dd-trace/src/noop/proxy.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const NoopTracer = require('./tracer')
+
+const noop = new NoopTracer()
+
+class Tracer {
+  constructor () {
+    this._tracer = noop
+  }
+
+  init () {
+    return this
+  }
+
+  use () {
+    return this
+  }
+
+  trace (name, options, fn) {
+    if (!fn) {
+      fn = options
+      options = {}
+    }
+
+    if (typeof fn !== 'function') return
+
+    options = options || {}
+
+    return this._tracer.trace(name, options, fn)
+  }
+
+  wrap (name, options, fn) {
+    if (!fn) {
+      fn = options
+      options = {}
+    }
+
+    if (typeof fn !== 'function') return fn
+
+    options = options || {}
+
+    return this._tracer.wrap(name, options, fn)
+  }
+
+  setUrl () {
+    this._tracer.setUrl.apply(this._tracer, arguments)
+    return this
+  }
+
+  startSpan () {
+    return this._tracer.startSpan.apply(this._tracer, arguments)
+  }
+
+  inject () {
+    return this._tracer.inject.apply(this._tracer, arguments)
+  }
+
+  extract () {
+    return this._tracer.extract.apply(this._tracer, arguments)
+  }
+
+  scope () {
+    return this._tracer.scope.apply(this._tracer, arguments)
+  }
+
+  getRumData () {
+    return this._tracer.getRumData.apply(this._tracer, arguments)
+  }
+
+  setUser () {
+    return this._tracer.setUser.apply(this.tracer, arguments)
```

Comment:
In other methods it is `this._tracer`:
```suggestion
    return this._tracer.setUser.apply(this._tracer, arguments)
```

Reply:
This was already the case before, not sure why tests are passing. I would say to leave it as is for now and it can be fixed in another PR with a corresponding test. cc @simon-id

Reply:
Uh, the tests are passing because it actually works. The property `this.tracer` is actually defined and the only method I use is `tracer.scope()` which is proxied too. I guess I must have forgot the underscore and not notice since it works exactly the same. I'll make another PR to clean that up

Reply:
This is the only occurence, you can accept ugaitz suggestion, no need for a test since nothing was broken